src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+
+public class Application {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[ARRAY_SIZE];
+        for (int i = 0; i < figures.length;i++) {
+            if (i >= figures.length / 2) {
```

Comment:
It is a bit complicated if statement, it could be simplified and you can avoid using continue;